CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ Deprecations and compatibility notes:
 
 - The `GeoSeries.select` method wrapping the pandas `Series.select` method has been removed.
   The upstream method no longer exists in all supported version of pandas (#3394).
+- The deprecated `GeoSeries.geom_almost_equals` method has been removed. Use
+  `GeoSeries.geom_equals_exact` instead (#3522).
```

Comment:
```suggestion
- The deprecated `geom_almost_equals` method has been removed. Use
  `geom_equals_exact` instead (#3522).
```
`geom_almost_equals` is defined on base.py, so perhaps GeoSeries is overly specific? Fine either way.